devtools/common/src/main/filtered/extensions.json

```diff
@@ -601,5 +601,14 @@
     "groupId": "io.quarkus",
     "artifactId": "quarkus-vertx",
     "guide": "https://quarkus.io/guides/using-vertx"
+  },
+  {
+    "name": "Scala",
+    "labels": [
+      "scala"
+    ],
+    "groupId": "io.quarkus",
+    "artifactId": "quarkus-scala",
+    "guide": "https://quarkus.io/guides/scala"
```

Comment:
Let's remove this last entry as we don't have a guide yet.

Reply:
Removed the entry and squashed them.